src/enforcementLevel.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2016-2019, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { EnforcementLevel } from "@pulumi/policy";
+
+/** @internal */
+export const defaultEnforcementLevel: EnforcementLevel = "mandatory";
```

Comment:
I believe we talked about making them "advisory" by default"... im cool with either way just want to know why we changed

Reply:
I just matched how we were exporting these [currently](https://github.com/pulumi/pulumi-awsguard/blob/d7d217b774bc2077789637a70f1ab0a85d980fea/src/compute.ts#L18-L24), but happy to make `"advisory"` the default. Chris had mentioned that in his PR as well.